pkg/network/tracer/tracer_windows.go

```diff
@@ -301,3 +327,28 @@ func newUSMMonitor(c *config.Config, dh driver.Handle) usm.Monitor {
 	monitor.Start()
 	return monitor
 }
+
+func (t *Tracer) addProcessInfo(c *network.ConnectionStats) {
+	if t.processCache == nil {
+		return
+	}
+
+	c.ContainerID.Source, c.ContainerID.Dest = nil, nil
+
+	// on windows, cLastUpdateEpoch is already set as
```

Comment:
Nit: `LastUpdateEpoch`

Reply:
the member variable name is cLastUpdateEpoch.  Could you clarify what you're expecting here?

Reply:
It looked like a typo, but this is fine.